apps/ui/src/Communities/CommunityCard/CommunityCard.tsx

```diff
@@ -0,0 +1,72 @@
+import Card from '@mui/material/Card';
+import Box from '@mui/material/Box';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+
+import defaultTheme from '../../themes/default';
+
+import { UserStatus } from '@communecar/types';
+import { useState } from 'react';
+import { StatusButton } from './StatusButton';
+import { Grid } from '@mui/material';
+import { CommunityMembersDisplay } from './CommunityMembersDisplay';
+import { ClientCommunity } from '../../types/community-type';
+
+export interface CommunityCardProps {
+  community: ClientCommunity;
+  userStatus: UserStatus;
+}
+
+const CommunityCard: React.FC<CommunityCardProps> = ({
+  community,
+  userStatus,
+}) => {
+  const { name, description, picturesUrl } = community;
+  const [joined, setJoined] = useState(false);
+
+  //TODO - handleChangeStatus
+  console.log({joined});//TODO when the server ready
+  
+  return (
+    <Box sx={{ margin: '5%', width: '100%', maxWidth: 400 }}>
```

Comment:
Where did you get this `maxWidth` value from?

Reply:
?